tests/integration-tests/tests/iam/test_iam/test_iam_roles/HIT.update.ini

```diff
@@ -0,0 +1,29 @@
+[global]
```

Comment:
It seems to me that you don't need two files to test the update since the variables are the same, the files are identical. 
You can reuse the same template and re-render it with the new variables.